.changeset/young-rabbits-train.md

```diff
@@ -0,0 +1,16 @@
+---
+"node-app-pages": patch
+"pages-functions-app": patch
+"pages-workerjs-app": patch
+"remix-pages-app": patch
```

Comment:
let's not add fixtures in changesets
```suggestion
```

Reply:
resolved: https://github.com/cloudflare/wrangler2/pull/1691/commits/fb1d32f80224fd1f9c2d1e6d554fe19939cb6f58